src/explore-education-statistics-common/src/components/Modal.tsx

```diff
@@ -81,6 +81,7 @@ const Modal = ({
           data-testid="modal-underlay"
         >
           <Dialog.Content
+            aria-describedby={undefined}
```

Comment:
@Guy-HiveIT I noticed this while checking something else - the modal was adding a `aria-describedby` reference to an id that doesn't exist.